processing/src/main/java/org/apache/druid/query/DruidProcessingIndexesConfig.java

```diff
@@ -31,37 +31,24 @@ public class DruidProcessingIndexesConfig
   @JsonProperty
   private final double skipValueRangeIndexScale;
 
-  @JsonProperty
-  private final double skipValuePredicateIndexScale;
-
   @JsonCreator
   public DruidProcessingIndexesConfig(
-      @JsonProperty("skipValueRangeIndexScale") @Nullable Double skipValueRangeIndexScale,
-      @JsonProperty("skipValuePredicateIndexScale") @Nullable Double skipValuePredicateIndexScale
```

Comment:
this parameter is for an easy way myself and other druid devs to experiment with stuff in case I didn't get the default scale value in the best spot; i don't really intend on cluster operators setting this value or for it to be documented. it controls the ratio of number of rows to be scanned vs computing indexes so effectively allowing adjusting the tuning performance of filter operations without code changes, but ideally it will eventually be removed